src/commands/dev/index.ts

```diff
@@ -56,6 +57,17 @@ export default async function main(ctx: NowContext) {
     return 2;
   }
 
+  const pkg = await readPackage();
+  if (pkg) {
+    // @ts-ignore
+    const { scripts } = pkg;
+
+    if (scripts.dev && scripts.dev === 'now dev') {
```

Comment:
```suggestion
    if (scripts.dev && (scripts.dev === 'now dev' || scripts.dev.startsWith('now dev ') || scripts.dev.endsWith(' now dev') || scripts.dev.includes(' now dev '))) {
```
We need to check if the script includes `now dev`, since it could also be `ENV=test now dev` or `custom.js && now dev` etc.

Reply:
Why not just `.includes()`? It would catch the starts/ends with scenarios as well.

```suggestion
    if (scripts.dev && scripts.dev.includes('now dev')) {
```

Reply:
I think the problem @TooTallNate is that your solution would match the corner case of `"dev": "snow devil"` 👿

Reply:
🤣 How about this then?

```suggestion
    if (scripts.dev && /\bnow\bdev\b/.test(scripts.dev)) {
```

Reply:
`/\bnow\bdev\b/.test('now dev')` is false 🤔

Reply:
Sorry, it should be `/\bnow\b\W+\bdev\b/.test('now dev')`

Reply:
@TooTallNate updated